src/XRP/issued-currency-client.ts

```diff
@@ -124,4 +135,66 @@ export default class IssuedCurrencyClient {
     })
     return trustLines
   }
+
+  /**
+   * Creates a trustline between this XRPL account and an issuer of an IssuedCurrency.
+   *
+   * @see https://xrpl.org/trustset.html
+   *
+   * TODO (tedkalaw): Implement qualityIn/qualityOut.
```

Comment:
why not yet?

Reply:
i don't really get how they work yet and seems like an easy add later. if i do this + authorize then we have everything we need to test things e2e

Reply:
yeah fair, i'm also having a hard time finding documentation that actually explains what these are used for in practice... maybe you can just make an Asana task to make sure you don't forget about adding it later